gqueries/modules/factsheet/technologies_section/biomass/factsheet_demand_of_biomass_other.gql

```diff
@@ -1,11 +1,6 @@
 - query =
-      SUM(V(
-            agriculture_final_demand_wood_pellets,
-            other_final_demand_wood_pellets,
-            industry_final_demand_wood_pellets,
-            bunkers_final_demand_bio_kerosene,
-            demand
-          )
-      )
+      SUM(Q(factsheet_supply_woody_biomass),
```

Comment:
I understand why you did this @michieldenhaan but it will obscure any errors as this will always correct for missing energy...